pandas/io/json/json.py

```diff
@@ -28,7 +28,7 @@
 # interface to/from
 def to_json(path_or_buf, obj, orient=None, date_format='epoch',
             double_precision=10, force_ascii=True, date_unit='ms',
-            default_handler=None, lines=False, compression=None,
+            default_handler=None, lines=False, compression='infer',
```

Comment:
Not sure where to update the to_json docs... didn't see a docstring in this function.

Reply:
https://github.com/pandas-dev/pandas/blob/322dbf4f812f47d4eaee5d8afa10e17acf6dbc67/pandas/core/generic.py#L1905